Architect-Coders/MU8
# Mu8/alice/issue5 create data module

**Create Data Module:** Issue #5   .

- Add create data module.

**Testing Done:**

* On real device: Huawei 20P Lite.

**How to test:**

*Create a **DummyClass** in the data module
*Add module dependency `implementation project (": data")` into **build.gradle(app)**
*In the App module (presentation) of the MainActivity.kt class add an instance of the **DummyClass** and verify that the class is imported correctly.

**Signed-off-by:**

* [aliceresponde](https://github.com/aliceresponde)  aliceresponde@gmail.com
